src/utils/utils.tsx

```diff
@@ -73,15 +73,12 @@ const getModifiedList = <T extends { name: string }>(items: T[], selected: strin
   }
 };
 
-export const getModifiedSystemsList = (systems: System[], selected: string | null) =>
-  getModifiedList(systems, selected);
-
-export const getModifiedFaultTreesList = (faultTrees: FaultTree[], selected: string | null) => {
-  if(!faultTrees || !selected)
-    return faultTrees;
-  return faultTrees.filter(f => f?.system?.iri == selected);
-}
+export const getModifiedSystemsList = (systems: System[], selected: System) => getModifiedList(systems, selected);
 
+export const getModifiedFaultTreesList = (faultTrees: FaultTree[], selected: System) => {
```

Comment:
We have getModifiedSystemsList and getModifiedFaultTreesList (CC @Kasmadei ): 
-- basically the same name without any comment on what it does, but they do fundamentally different things :(. This is confusing and can lead to errors easily.
-- getModifiedSystemsList -- reorders elements based on selected one
-- getModifiedFaultTreesList -- filter elements based on an attribute

Reply:
I will create ticket for that.

Reply:
Will be resolved in https://github.com/kbss-cvut/fta-fmea-ui/pull/422